app/controllers/search_queries_controller.rb

```diff
@@ -0,0 +1,31 @@
+class SearchQueriesController < ApplicationController
+  
+  # before_action :authenticate_user!
+
+  def index
+    @search_query = SearchQuery.find_by(user_id: current_user.id) if current_user
+  end
+
+  def new
+    @search_query = SearchQuery.new
+  end
+
+  def edit
+    @search_query = SearchQuery.find(search_query_params)
+  end
+
+  def create
+    query = SearchQuery.create(user: current_user)
+    query.update_attributes(search_query_params)
+    query.save
```

Comment:
I'd put a new line here.  A good rule of thumb is never have more than 3 lines in a row and group things as you would a paragraph.